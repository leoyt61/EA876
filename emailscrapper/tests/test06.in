emails123@99taxis.com podem ter numeros e ate comecar com numeros! Mesmo usernames como 99taxis@99taxis.com sao possiveis!
