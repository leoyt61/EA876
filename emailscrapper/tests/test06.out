emails123@99taxis.com
99taxis@99taxis.com
